tests/crypto/src/common_test.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <stdio.h>
+#include <stdarg.h>
+#include <string.h>
+#include <stdint.h>
+#include <stddef.h>
+#include <misc/printk.h>
+
+#include <sys/util.h>
+#include <sys_clock.h>
+
+#include "common_test.h"
+
+volatile uint32_t t_start;
+volatile uint32_t t_stop;
```

Comment:
Where are these read?

Also, it should be commented why they are volatile.

E.g. 

// t_start is written to by thread A, and read by thread B.

Reply:
Unused, removed in 7aee7a9